src/main/java/gameframework/drawing/GameCanvasDefaultImpl.java

```diff
@@ -36,8 +39,11 @@ public void drawFullSizeImage(Graphics graphics, Image image) {
 	}
 
 	@Override
-	public void drawFullSizeImage(Image image) {
```

Comment:
when you change the signature of a method, please try to keep the old signature in parallel for backward compatibility. You may want to add the `@Deprecated` annotation to old code.